lib/galaxy/datatypes/converters/interval_to_bgzip_converter.xml

```diff
@@ -12,15 +12,15 @@
         '$input1' '$output1'
     </command>
     <inputs>
-        <param format="interval" name="input1" type="data" label="Choose Interval file"/>
+        <param format="interval,encodepeak" name="input1" type="data" label="Choose Interval file"/>
```

Comment:
This shouldn't be needed, since `ENCODEPeak` is a subclass of `Interval`.

Reply:
done